Server/uploadpackage.R

```diff
@@ -20,11 +20,10 @@ observeEvent(
   )
 )
 
-#' Save all the uploaded packages, marking them as 'new', 'not found', or
-#' 'duplicate'.
-observe({
-  req(input$uploaded_file)
-  
+#' Save all the uploaded packages, 
+#' marking them as 'new', 'not found', or 'duplicate'.
+observeEvent(input$uploaded_file, {
```

Comment:
Hi @Robert-Krajcik. Why won't an `observe()` work here? Just curious. It looks like the only inputs are input$uploaded_file, right?